config/crds/azureprovider_v1alpha1_azuremachineproviderspec.yaml

```diff
@@ -33,6 +36,9 @@ spec:
           - version
           type: object
         kind:
+          description: 'Kind is a string value representing the REST resource this
+            object represents. Servers may infer this from the endpoint the client
+            submits requests to. Cannot be updated. In CamelCase. More info: https://git.k8s.io/community/contributors/devel/api-conventions.md#types-kinds'
```

Comment:
Same as previous.

Reply:
This is actually generated from `k8s.io/apimachinery/pkg/apis/meta/v1`: `TypeMeta.Kind`